cypress/component/user/Hackpack.cy.jsx

```diff
@@ -0,0 +1,52 @@
+import Hackpack from "@/components/user/Hackpack";
+describe("HackPacks", () => {
+  it("HackPack Components", () => {
+    const text = "Hackpack";
```

Comment:
most of these variables can be placed directly where they r being used, making multiple variables makes it harder to track everything